jobs/web/spec

```diff
@@ -240,6 +240,15 @@ properties:
          ...
          -----END PUBLIC KEY-----
 
+  config_rbac:
+    env_file: CONCOURSE_CONFIG_RBAC
+    description: |
+      YAML file content to customize RBAC role-action mapping.
+    example: |
```

Comment:
@evanchaoli since you added the code change, do you have any examples of interesting use cases that might be good to add as part of the packaging documentation? We just picked random permissions to show that the values for each role should be a list.

Reply:
@deniseyu  Yes, a real use case is, on our cluster, we have on-duty folks who are responsible for monitoring pipeline status. They are "team members", but we still want them to be able to hijack into containers to debug failures. But the hijack-container permission is only on the role "team-member", so that we'll change hijack-container to pipleine-operator on our clusters.